bin/add-idam-clients.sh

```diff
@@ -14,4 +14,6 @@ ${dir}/utils/idam-create-service.sh "am_role_assignment" "am_role_assignment" "a
 
 ${dir}/utils/idam-create-service.sh "ccd_data_store_api" "ccd_data_store_api" "idam_data_store_client_secret" "http://ccd-data-store-api/oauth2redirect" "false" "profile openid roles manage-user"
 
-${dir}/utils/idam-create-service.sh "ccd_case_disposer" "ccd_case_disposer" "idam_ccd_case_disposer_secret" "http://ccd-case-disposer/oauth2redirect" "false" "profile openid roles"
\ No newline at end of file
+${dir}/utils/idam-create-service.sh "ccd_case_disposer" "ccd_case_disposer" "idam_ccd_case_disposer_secret" "http://ccd-case-disposer/oauth2redirect" "false" "profile openid roles"
+
+${dir}/utils/idam-create-service.sh "ccd_next_hearing_updater_service" "ccd_next_hearing_updater_service" "AAAAAAAAAAAAAAAA" "http://ccd-next-hearing-updater-service/oauth2redirect" "false" "profile openid roles"
```

Comment:
As I explained to you last week, we've gone with `ccd_next_hearing_updater` elsewhere and not `ccd_next_hearing_updater_service`.  The thinking is that every microservice is a service thus the word `service` in the name is rather superfluous.  As you can see here:
 -  for data store, it's just `ccd_data_store_api`
 - and case disposer, it's `ccd_case_disposer`

Reply:
I have changed with @danlysiak and this is the recommend name , I will remove the service and name like this one ccd_next_hearing_date_updater

Reply:
This has been updated 👍